cba_settings.sqf

```diff
@@ -218,7 +211,6 @@ force acex_headless_enabled = false;
 force acex_headless_endMission = 0;
 force acex_headless_log = false;
 force acex_sitting_enable = true;
-force acex_volume_enabled = true;
```

Comment:
da is der default false - um das bisherige Verhalten nicht zu ändern (außer für die die es wirklich wollen) würd ich hier lediglich das "force" rausnehmen wollen